app/submitter/converter.py

```diff
@@ -192,7 +193,7 @@ def _encode_value(value):
     if isinstance(value, int):
         return str(value)
     elif isinstance(value, datetime):
-        return value.strftime(settings.SDX_DATE_FORMAT)
+        return value.strftime(SDX_DATE_FORMAT)
```

Comment:
Im not sure that this line ever gets hit anymore? as we no longer store datetime objects in the answer store.